operations/operations/zuul-ci-cd-quickstart-user-guide.md

```diff
@@ -0,0 +1,236 @@
+# Zuul users guide
+
+## Prerequisites
+
+1. Repository is known by [SCS zuul](https://zuul.scs.community)
```

Comment:
I see no reason why Zuul isn't spelled with a capital Z, as is the case in the Zuul dashboard.

Reply:
all occurrences that references to the name Zuul starts with a capital Z